docs/framework/solid/guides/form-groups.md

```
@@ -0,0 +1,178 @@
+  name="step1"
+  validators={{
+    onChange: z.object({
+      name: z.string().min(2),
+    }),
+  }}
+/>
+```
+
+> The reason we don't use the full path names for fields is so that you can compose your schemas like so:
+>
+> ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)